File: go/961_N-Repeated_Element_in_Size_2N_Array/961_N-Repeated_Element_in_Size_2N_Array.go
func repeatedNTimes(nums []int) int {
    m := make(map[int]int)
    n := len(nums) / 2
    for _, v := range nums {
        m[v] += 1
    }
    result := -1
    for k := range m {
        if m[k] == n {
            result = k
            break
        }
    }
    return result
}

File: java/961_N-Repeated_Element_in_Size_2N_Array/961_N-Repeated_Element_in_Size_2N_Array.java
class Solution {
    public int repeatedNTimes(int[] nums) {
        int n = nums.length / 2;
        HashMap<Integer, Integer> map = new HashMap<Integer, Integer>();
        for (int i : nums) {
            map.merge(i, 1, Integer::sum);
        }

        int result = -1;
        for (int key : map.keySet()) {
            if (map.get(key) == n) {
                result = key;
                break;
            }
        }
        return result;
    }
}
